StRoot/StFwdTrackMaker/include/Tracker/TrackFitter.h

```diff
@@ -769,11 +828,12 @@ class TrackFitter {
 
         // get the seed info from our hits
         TVector3 seedMom, seedPos;
```

Comment:
Since you are initializing these below... you can make these static variables and save the overhead of registering them with ROOT every time the ctor is called.